src/molluscpalette.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <vector>
+#include <tuple>
+#include <random>
+
+#include <QColor>
+
+#include "mollusc.hpp"
+
+class MolluscPalette
+{
+public:
+    MolluscPalette(QString dataPath);
+    ~MolluscPalette();
+
+    std::vector<Mollusc>* getMolluscs() const;
```

Comment:
maybe return a const vector * here, or is there a use case for manipulating the molluscs?